pagopa/pn-frontend-e2e
# PN-9237-e-2-e-mittente-sezione-stato-piattaforma-disservizi-in-corso

## Short description
There isn't a test related to the **PN-9237** as you can see in the [confluence page](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868778282/Mittente+-+Mapping+test-scenari)

## List of changes proposed in this pull request
- added new test file `023_a23_visualizzazioneDIsserviziInCorso.feature`
- added new step file class `DisserviziAppPATest` with new feature step:
```
- Nella pagina Piattaforma Notifiche selezionare la voce 'stato della piattaforma
- Si visualizza correttamente la pagina dello 'stato della piattaforma' di mittente
- Si visualizza correttamente la tabella dei disservizi
- Si visualizza un record in elenco relativo ad un disservizio ancora in corso
```
- added related checks and methods
- added new files for future api integration


## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_MittenteVisualizzazioneDisservizi` and verify if the test works fine.
